src/generator.csproj

```diff
@@ -99,6 +99,7 @@
     <Compile Include="..\src\bgen\BindingTouch.cs" />
     <Compile Include="..\src\bgen\Enums.cs" />
     <Compile Include="..\src\bgen\ExtensionMethods.cs" />
+    <Compile Include="..\src\bgen\Frameworks.cs" />
```

Comment:
nit but should this be below `<Compile Include="..\src\bgen\Filters.cs" />` to maintain alphabetical order?